packages/example/src/comments/comments.queries.ts

```diff
@@ -25,7 +25,7 @@ const getAllCommentsIR: any = {"name":"GetAllComments","params":[{"name":"id","r
 /**
  * Query generated from SQL:
  * ```
- * SELECT * FROM book_comments WHERE id = :id! OR user_id = :id
+ * SELECT * FROM book_comments WHERE id = :id! OR user_id = :id                                      
```

Comment:
These spaces could be an artifact of my eccentric environment; I am on a windows machine using the windows subsystem for linux + windows docker desktop WSL integration.